core/templates/core/index.html

```diff
@@ -22,25 +22,25 @@
 </script>
 {% endblock %}
 
-
 {% block content %}
+<div class="h-screen w-screen">
+    <nav class="flex flex-wrap justify-between items-center mx-auto w-screen p-2 border-gray-800 bg-gradient-to-b from-indigo-700 via-indigo-800 to-indigo-900">
+            <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="flex items-center text-gray-300 transition-colors duration-300 ml-5" target="_blank">
+                <i class="fa-brands fa-github fa-3x mr-5 hover:text-blue-100"></i>
```

Comment:
If whole image + text is link, why `hover:text-blue-100` is only for image?

Reply:
caus thats what i was told to do 
https://github.com/Gwardia-Czapli/GwardiaHub/pull/26#discussion_r1572260888

Reply:
I thought that it looks better when only icon is highlighted

Reply:
Maybe... But now when you hover on text nothing changes, even icon.